### PR TITLE
feat: update admin dashboard docs

### PR DESCRIPTION
**Pull request by frdomovic**
# feat: update admin dashboard docs

## Summary
Updated how to start admin dashboard and provided detailed explanations and picture of all functionalities.

https://github.com/user-attachments/assets/84ae3f96-fa20-4b35-8e83-b557043655e4

### CODE REVIEW

File: docs/03-getting-started/02-admin-dashboard.mdx [hunk 1]
```diff
@@ -12,44 +12,217 @@ overseeing and controlling various aspects of your node infrastructure.
...
+#### Start new context
+
+To start new context press the "Start new context" button on previous page.
+
+You can select application with which you want to create a new context or you
+can upload a new application.
+
+![StartContextSelect](/admin-dashboard-functions/19start-context-select.png)
+
+You can add initialization parameters if your has initialization requirements.
```
Comment (xilosada) [inline docs/03-getting-started/02-admin-dashboard.mdx:210]: Can you clraify this part?

File: docs/03-getting-started/02-admin-dashboard.mdx [hunk 2]
```diff
@@ -12,44 +12,217 @@ overseeing and controlling various aspects of your node infrastructure.
...
+
+```bash title="Terminal"
+cd node-ui
+```
+
+Install dependencies and build the project.
+
+```bash title="Terminal"
+pnpm install && pnpm build
+```
```
Comment (alenmestrov) [inline docs/03-getting-started/02-admin-dashboard.mdx:31]: this is not needed

File: docs/03-getting-started/02-admin-dashboard.mdx (hunk 1 above)
Comment (frdomovic) [reply docs/03-getting-started/02-admin-dashboard.mdx]: Updated

File: docs/03-getting-started/02-admin-dashboard.mdx (hunk 2 above)
Comment (frdomovic) [reply docs/03-getting-started/02-admin-dashboard.mdx]: Updated

**Pull request merged**